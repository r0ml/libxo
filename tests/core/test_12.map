# test map
tagÜÖÄ=tagger
cölor=color
säfe=safe

# the other way
2by4=tagÜÖÄ1
2by4=tagÜÖÄ2
2by4=tagÜÖÄ3
animal=jack-not-name