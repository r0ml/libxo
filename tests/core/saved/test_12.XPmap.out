<top>
  <data>
    <jack-not-name>fish</jack-not-name>
    <jack-not-name>fish</jack-not-name>
    <thing>
      <name>thing</name>
      <color>green</color>
      <time>2:15</time>
      <hand>left</hand>
      <color>blue</color>
      <time>3:45</time>
    </thing>
    <thing>
      <name>thing</name>
      <color>green</color>
      <time>2:15</time>
      <hand>left</hand>
      <color>blue</color>
      <time>3:45</time>
    </thing>
    <thing>
      <name>thing</name>
      <color>green</color>
      <time>2:15</time>
      <hand>left</hand>
      <color>blue</color>
      <time>3:45</time>
    </thing>
    <thing>
      <name>thing</name>
      <color>green</color>
      <time>2:15</time>
      <hand>left</hand>
      <color>blue</color>
      <time>3:45</time>
    </thing>
    <thing>
      <name>thing</name>
      <color>green</color>
      <time>2:15</time>
      <hand>left</hand>
      <color>blue</color>
      <time>3:45</time>
    </thing>
    <thing>
      <name>thing</name>
      <color>green</color>
      <time>2:15</time>
      <hand>left</hand>
      <color>blue</color>
      <time>3:45</time>
    </thing>
    <thing>
      <name>thing</name>
      <color>green</color>
      <time>2:15</time>
      <hand>left</hand>
      <color>blue</color>
      <time>3:45</time>
    </thing>
    <thing>
      <name>thing</name>
      <color>green</color>
      <time>2:15</time>
      <hand>left</hand>
      <color>blue</color>
      <time>3:45</time>
    </thing>
    <thing>
      <name>thing</name>
      <color>green</color>
      <time>2:15</time>
      <hand>left</hand>
      <color>blue</color>
      <time>3:45</time>
    </thing>
    <thing>
      <name>thing</name>
      <color>green</color>
      <time>2:15</time>
      <hand>left</hand>
      <color>blue</color>
      <time>3:45</time>
    </thing>
    <tagÜÖÄ3>
      <_4x4>truck</_4x4>
      <_2morrow>tomorrow</_2morrow>
    </tagÜÖÄ3>
    <tagger>
      <color>blue</color>
      <safe>yes</safe>
    </tagger>
  </data>
</top>
